www/scripts/codecheckerviewer/CheckerStatistics.js

```diff
@@ -208,4 +298,4 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
       this.addChild(this._checkerStatistics);
     }
   });
-});
+});
```

Comment:
I think this change is unneeded and usually, it is good to preserve the last end of line in files.